src/IRrecv.cpp

```diff
@@ -427,17 +432,18 @@ bool IRrecv::matchAtLeast(uint32_t measured, uint32_t desired,
   DPRINT(". Matching: ");
   DPRINT(measured);
   DPRINT(" >= ");
-  DPRINT(ticksLow(std::min(desired, TIMEOUT_MS * 1000), tolerance));
+  DPRINT(ticksLow(std::min(desired, (uint32_t) MS_TO_USEC(irparams.timeout)),
```

Comment:
No. Lint and compiler error/warnings otherwise. The later parameter defaults to an int.